.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+This repo is not closely monitored by product and development teams.
```

Comment:
Oh actually wait, one more thing. The point isn't about whether it's monitored or not, but rather about how to get issues resolved most quickly. I think this text might represent that better:

> To address your issues more quickly, please open the issue in the most appropriate repository. These are some of the most common repos: